package.json

```diff
@@ -78,7 +78,7 @@
     "postcss-loader": "1.2.1",
     "prompt": "1.0.0",
     "react-addons-test-utils": "15.4.1",
-    "react-hot-loader": "^1.3.1",
+    "react-hot-loader": "3.0.0-beta.6",
```

Comment:
Yeah, now that we've already pulled in 3.0.0-beta.6, let's just leave it. It's been out there for a few days without anyone reporting issues and I've been told it's quite stable. Not ideal, but at this point I'd say it's better to leave it than churn further on a rollback. Thanks for your work on the RR4 conversion!